index.js

```diff
@@ -36,42 +38,42 @@ module.exports = function (options) {
     : typeof options.threshold === 'string' ? bytes(options.threshold)
     : 1024
 
-  return function* compress(next) {
-    this.vary('Accept-Encoding')
-
-    yield* next
+  return (ctx, next) => {
```

Comment:
Was following style in koajs/koa, I think, but might opt to not make this one an arrow in order to maintain the name - for stack purposes